src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,20 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: child.props.nestedItems.reduce(this._hasSelectedDescendant, false),
```

Comment:
What about adding a property to turn on/off this feature with a default to false to no introduce a breaking change? (it's overriding the `initiallyOpen` property that people may have defined on their `ListItem`)